zzk/service/hoststate.go

```diff
@@ -397,8 +397,10 @@ func (l *HostStateListener) setInstanceState(containerExit <-chan time.Time, ssd
 			// container is attached and not paused, so pause the container
 			if err := l.handler.PauseContainer(serviceID, instanceID); err != nil {
 				logger.WithError(err).Error("Could not pause container")
-				l.cleanUpContainers([]string{stateID}, true)
-				return nil, nil, false
+				l.handler.ResumeContainer(serviceID, instanceID)
+				select{}
```

Comment:
If I return error here, backup won't interrupt, so I'm not sure how to stop backup on errors.
Also, the main issue for me is that I can't stop/restart service service after pausing failure. Service is just can't be stopped.
It stuck in `Stopping` state when I hit Stop button.
I tried updating service state in zk here, but it doesn't help.
I don't know what to do to stop service after pausing failure, also I don't know how to interrupt backup on error from this place without select{} or sleep.